tools/lsp/preview.rs

```diff
@@ -119,7 +123,8 @@ pub fn config_changed(
             let current = cache.current.clone();
             let ui_is_visible = cache.ui_is_visible;
             drop(cache);
-            if ui_is_visible {
+
+            if ui_is_visible && !current.path.as_os_str().is_empty() {
```

Comment:
Note: since load_preview's first line is to check if the path is empty, this is not needed.  (but also doesn't hurt)
(comment also valid for other use of load_preview.  But this is fine to leave as is)